### DISCUSSION TITLE
Crop Mode

### DISCUSSION THREAD

**Discussion by andi-makes** [SchmarrnDevs/Lighty#11] [Ideas and Suggestions]
Highlight Farmland where the light level is not optimal.

How would the highlight look like though? Ideas are welcome!

**Comment by mhmatthewhugley** [comment]
Pressure plate highlight or maybe slab of something.

**Comment by Aferallan** [comment]
Since I don't imagine it will be used very often, I don't think something aesthetically pleasing is necessary, so a block in the style of the carpet overlay would be very useful.

**Comment by andi-makes** [comment]
With v2.2.0, there is a Farmland mode!
